src/go/bootstrap/tracing_test.go

```diff
@@ -429,7 +429,7 @@ func runTest(_ *testing.T, shouldRunServer bool, f func()) {
 	if shouldRunServer {
 		// Run a mock server and point injected client to mock server
 		mockMetadataServer := util.InitMockServerFromPathResp(map[string]string{
-			util.ProjectIDSuffix: fakeMetadataProjectId,
+			util.ProjectIDPath: fakeMetadataProjectId,
```

Comment:
FYI, you'll have to fix some merge conflicts :)

Reply:
ACK